rllib/algorithms/bc/bc.py

```diff
@@ -166,34 +168,51 @@ def training_step(self) -> ResultDict:
                         max_env_steps=self.config.train_batch_size,
                     )
 
-                train_batch = train_batch.as_multi_agent()
-                self._counters[NUM_AGENT_STEPS_SAMPLED] += train_batch.agent_steps()
-                self._counters[NUM_ENV_STEPS_SAMPLED] += train_batch.env_steps()
+                # TODO (sven): Use metrics API as soon as we moved to new API stack
```

Comment:
Is this one not using the `MetricsLogger`, yet? I use it in my overhaul of offline RL

Reply:
Yes, you should. All good there. :)

But the hybrid API stack still goes through the `summarize_episodes` utility inside `algorithm.py`, which operates on the old `RolloutMetrics` objects returned by RolloutWorkers.